.github/workflows/continuous-integration.yml

```
@@ -27,34 +28,27 @@ jobs:
           - "8.2"
           - "8.3"
           - "8.4"
-        stability:
-          - "stable"
         symfony-version:
-          - "6.4.*"
-          - "7.2.*"
-          - "7.3.*"
+          - "locked"
         driver-version:
           - "stable"
         dependencies:
           - "highest"
         include:
+          # Tests the lowest set of dependencies
           - dependencies: "lowest"
-            os: "ubuntu-24.04"
             php-version: "8.1"
             driver-version: "1.21.0"
-            stability: "stable"
-            symfony-version: "6.4.*"
-          - dependencies: "highest"
-            os: "ubuntu-24.04"
-            php-version: "8.2"
-            driver-version: "mongodb/mongo-php-driver@v2.x"
-            stability: "dev"
-            symfony-version: "7.2.*"
-        exclude:
-          - php-version: "8.1"
-            symfony-version: "7.2.*"
-          - php-version: "8.1"
-            symfony-version: "7.3.*"
+
+          # Test LTS
+          - symfony-require: "6.4.*"
+            php-version: "8.4"
+          - symfony-require: "7.4.*"
```

The build uses `symfony-version`:

```suggestion
          - symfony-version: "6.4.*"
            php-version: "8.4"
          - symfony-version: "7.4.*"
```

---

That's a mix with [DoctrineBundle](https://github.com/doctrine/DoctrineBundle/blob/3.0.x/.github/workflows/continuous-integration.yml)
But in [MongoDB ODM](https://github.com/doctrine/mongodb-odm/blob/2.15.x/.github/workflows/continuous-integration.yml) we use `symfony-version`.